Makefile

```diff
@@ -1,12 +1,12 @@
 PYTEST = pytest -v
-COV_FLAGS = --cov-fail-under=93.5 --cov=codemodder --cov=core_codemods
+COV_FLAGS = --cov=codemodder --cov=core_codemods
 XDIST_FLAGS = --numprocesses auto
 
 test:
-	COVERAGE_CORE=sysmon ${PYTEST} ${COV_FLAGS} tests ${XDIST_FLAGS}
+	COVERAGE_CORE=sysmon ${PYTEST} ${COV_FLAGS} tests ${XDIST_FLAGS} && coverage json && coverage-threshold
 
 integration-test:
-	${PYTEST} integration_tests ${XDIST_FLAGS}
+	COVERAGE_CORE=sysmon ${PYTEST} integration_tests --cov=core_codemods ${XDIST_FLAGS}  && coverage json && coverage-threshold --line-coverage-min 80
```

Comment:
My suggestion is to do away with overall `line-coverage` here (and just keep the per-file one). 

Looking at the failed run in the previous commit, the line coverage for integration tests varies wildly and this will have to be tweaked a lot. Also, since we are already enforcing this for unit tests, it feels redundant here.

Reply:
The overall line coverage for integration tests doesn't vary, it's always above 80% which is much lower than unit tests. And that of course makes sense since integration tests don't intend to cover all paths.

That being said, I agree total line coverage doesn't make much sense for integration tests, but given how `coverage-threshold` is configured, I kinda have to pass in `--line-coverage-min #` because it first uses whatever I configured in pyproject.toml, which is the 93%+ for unit tests. Passing it as a CLI flag overrides that. Maybe it makes sense to just set it to `--line-coverage-min 0`? But that at point, 80 is just as well, all integration test runs I see are 83%+

Reply:
See here:
https://github.com/pixee/codemodder-python/actions/runs/8910019509/job/24468409979
Coverage can go as low as 50% and is achieved by at least one file:
```
 /opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/core_codemods/defectdojo/semgrep/avoid_insecure_deserialization.py      20     10    50%
```
That's what I meant by varying wildly.

Reply:
I think we're ok with that.